src/routes.tsx

```diff
@@ -39,18 +58,35 @@ export function Routes() {
 
   return (
     <HashRouter basename={"/"}>
-      <ConnectionProvider>
+      <ConnectionProvider endpoint={connectionEndpoint}>
         <WalletProvider wallets={wallets} autoConnect>
-          <AccountsProvider>
-            <MarketProvider>
-              <AppLayout>
-                <Switch>
-                  <Route exact path="/" component={() => <HomeView />} />
-                  <Route exact path="/faucet" children={<FaucetView />} />
-                </Switch>
-              </AppLayout>
-            </MarketProvider>
-          </AccountsProvider>
+          <TokensProvider>
+            <AccountsProvider>
+              <MarketProvider>
+                <AppLayout>
+                  <Switch>
+                    <Route exact path="/" component={() => <HomeView />} />
+                    <Route exact path="/faucet" children={<FaucetView />} />
+                    <Route
+                      exact
+                      path="/account-information"
+                      children={<AccountInformation />}
+                    />
+                    <Route
+                      exact
+                      path="/token-minting"
+                      children={<TokenMinting />}
+                    />
+                    <Route
+                      exact
+                      path="/send-solana"
+                      children={<SendSolana />}
+                    />
```

Comment:
What's the difference between doing `component={() => <HomeView />}` and `children={<AccountInformation />}`? It seems like providing a function could be better since it will avoid extra renders.